app/assets/javascripts/components/generic_object/main-custom-button-form.js

```diff
@@ -258,8 +283,14 @@ function mainCustomButtonFormController(API, miqService, $q, $http) {
         vm.customButtonModel.attribute_names,
         vm.customButtonModel.attribute_values);
 
+      if(vm.customButtonModel.uri_attributes.hosts == undefined || vm.customButtonModel.uri_attributes.hosts === "localhost" || vm.customButtonModel.uri_attributes.hosts == null) {
```

Comment:
missing spaces after if, and before { on line 288

also, no point testing *both* `vm.customButtonModel.uri_attributes.hosts == undefined` and `vm.customButtonModel.uri_attributes.hosts == null`. Those are exactly the same thing without `===`.

(And this should really be just `if (!vm.customButtonModel.uri_attributes.hosts || vm.customButtonModel.uri_attributes.hosts === 'localhost')` .. comparing against null or undefined is usually not what you want :).